### ISSUE TITLE
Survey form not valid

### ISSUE THREAD

**Issue by money8203** [ConsenSys-Academy/Blockchain-Developer-Bootcamp#295]
In Blockchain-Developer Bootcamp - - - >S00-intro - - - >L1-course-intro - - - > index.md
The survey form is invalid 
Showing page 404 !

**Comment by lonasozo** [comment]
Should be enough remove the link and add some reference like "Deprecated" near the link? Or make more sense to remove the entire "Live Presentations" section?

**Comment by money8203** [comment]
it will be good enough either to remove the damaged link and add some reference or replace old link woth new working link! removing the entire live presentation section won't be a good choice i think, what you say ?

**Comment by lonasozo** [comment]
I don't know, it seems something obsolete to me, @GuiBibeau what do you think?

**Comment by money8203** [comment]
I am still encountering the same problem but!

**Comment by lonasozo** [comment]
Yes it is broken, before changing it we need to understand how to proceed

**Comment by money8203** [comment]
yea sure, discuss it with the team members :)
